src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.field-wrapper {
+  margin-bottom: 10px;
+}
+
+.chapter-wrapper {
```

Comment:
I would suggest here to give a more descriptive name for this class, maybe something like .fieldset-wrapper or .form-section :)